enigma-js/src/Enigma.js

```diff
@@ -549,6 +549,20 @@ export default class Enigma {
     return emitter;
   }
 
+  /**
+   * Poll the ETH for a Task's status
+   *
+   * @param {Task} task - Task wrapper for contract deployment and compute tasks
+   * @return {Task} Task wrapper with updated ETH status.
+   */
+  async pollTaskETH(task) {
```

Comment:
I would suggest to add an optional parameter `interval` that defaults to `1000`, but allows the user to change the polling interval if needed.

Reply:
resolved in e06e8d7614014a9a604073b326773ad2411c536c